modules/SofaGeneralEngine/src/SofaGeneralEngine/PlaneROI.inl

```diff
@@ -400,7 +400,7 @@ void PlaneROI<DataTypes>::draw(const core::visual::VisualParams* vparams)
     vparams->drawTool()->disableLighting();
 
     const VecCoord* x0 = &f_X0.getValue();
-    const sofa::type::RGBAColor& color = sofa::type::RGBAColor::cyan();
+    constexpr const sofa::type::RGBAColor& color = sofa::type::RGBAColor::cyan();
```

Comment:
sometimes you remove the const, sometimes not (here is this case), is it normal?

Reply:
it depends if the type is a value or a const reference. Here I keep the const because it's a const reference. However, if it's a value, the const is redundant.

Reply:
👍